docs/api/paths/diagnostic/keyspace-docid-_all_channels.yaml

```
@@ -0,0 +1,37 @@
+# Copyright 2022-Present Couchbase, Inc.
+#
+# Use of this software is governed by the Business Source License included
+# in the file licenses/BSL-Couchbase.txt.  As of the Change Date specified
+# in that file, in accordance with the Business Source License, use of this
+# software will be governed by the Apache License, Version 2.0, included in
+# the file licenses/APL2.txt.
+parameters:
+  - $ref: ../../components/parameters.yaml#/db
+  - $ref: ../../components/parameters.yaml#/user-name
```

```suggestion
  - $ref: ../../components/parameters.yaml#/db
  - $ref: ../../components/parameters.yaml#/user-name
```

I think these need to be `keyspace` and `docid`?